src/redux/ducks/session/data.js

```diff
@@ -52,22 +52,21 @@ const createSessionSuccess = (payload) => ({type: SESSION_CREATE_SUCCESS, payloa
 const loadUserProfile = (profile: UserModel) => ({type: SESSION_PROFILE, profile});
 const destroySession = (next) => ({type: SESSION_DESTROY, next});
 
-const checkLOCControllers = (index, LOCCount, account) => {
-    return Promise.resolve(false);// todo: remove loc check
-    if (index >= LOCCount) {
-        return Promise.resolve(false);
-    }
-    return AppDAO.getLOCbyID(index).then(r => {
-        const loc = new LocDAO(r);
-        return loc.isController(account).then(r => {
-            if (r) {
-                return true;
-            } else {
-                return checkLOCControllers(index + 1, LOCCount, account);
-            }
-        });
-    });
-};
+// const checkLOCControllers = (index, LOCCount, account) => {todo: loc check isn't needed now
+//     if (index >= LOCCount) {
+//         return Promise.resolve(false);
+//     }
+//     return AppDAO.getLOCbyID(index).then(r => {
+//         const loc = new LocDAO(r);
+//         return loc.isController(account).then(r => {
+//             if (r) {
+//                 return true;
+//             } else {
+//                 return checkLOCControllers(index + 1, LOCCount, account);
+//             }
+//         });
+//     });
+// };
```

Comment:
Please don't write such comments here. Your commits in the future will show whether you fixed it or not.